Makefile

```diff
@@ -24,17 +24,17 @@ SERVER_DIR:=images/server
 AD_SERVER_DIR:=images/ad-server
 CLIENT_DIR:=images/client
 TOOLBOX_DIR:=images/toolbox
-SERVER_SRC_FILE:=$(SERVER_DIR)/Containerfile.fedora
+SERVER_SRC_FILE:=$(SERVER_DIR)/Containerfile.centos
 SERVER_SOURCES:=\
 	$(SERVER_DIR)/smb.conf \
 	$(SERVER_DIR)/install-packages.sh \
 	$(SERVER_DIR)/install-sambacc.sh
-AD_SERVER_SRC_FILE:=$(AD_SERVER_DIR)/Containerfile.fedora
+AD_SERVER_SRC_FILE:=$(AD_SERVER_DIR)/Containerfile.centos
 AD_SERVER_SOURCES:=\
 	$(AD_SERVER_DIR)/install-packages.sh \
 	$(AD_SERVER_DIR)/install-sambacc.sh
-CLIENT_SRC_FILE:=$(CLIENT_DIR)/Containerfile.fedora
-TOOLBOX_SRC_FILE:=$(TOOLBOX_DIR)/Containerfile.fedora
+CLIENT_SRC_FILE:=$(CLIENT_DIR)/Containerfile.centos
+TOOLBOX_SRC_FILE:=$(TOOLBOX_DIR)/Containerfile.centos
```

Comment:
These files don't exist yet.

Reply:
I changed the commit to only add a server centos flavor ...

Reply:
Now I added a second commit to add the client centos flavor